controllers/admin/AdminSelfUpgradeController.php

```diff
@@ -526,6 +528,35 @@ public function initContent()
                 ->getJson()
         );
 
+        $this->addNewUIAssets();
+
         return parent::initContent();
     }
+
+    /**
+     * @return void
+     */
+    private function addNewUIAssets()
+    {
+        if (!empty($_ENV['AUTOUPGRADE_DEV_WATCH_MODE']) && $_ENV['AUTOUPGRADE_DEV_WATCH_MODE'] === '1') {
```

Comment:
Not sure if this should be in the module files guys, isn't there some other way to run the dev environment?

Reply:
Unfortunately the Vite dev mode (_dev folder) has no influence on the symfony part, that's why I decided to add an env file to define it manually. But this has no impact on production.